src/test/java/fr/esgi/al5_2/Tayarim/ProprietaireControllerTest.java

```diff
@@ -36,279 +37,349 @@
+        .id(2L)
+        .nom("Jhonny")
+        .prenom("Pasdoe")
+        .email("test2@gmail.com")
+        .numTel("0612345679")
+        .dateInscription(localDateTime)
+        .logements(null)
+        .build();
+    proprietaireDTOS = List.of(proprietaireDTO, proprietaireDTO2);
+    proprietaireDTOWithLogement = ProprietaireDTO.builder()
+        .id(1L)
+        .nom("Ferreira")
+        .prenom("Mathieu")
+        .email("test@gmail.com")
+        .numTel("0612345678")
+        .dateInscription(localDateTime)
+        .logements(List.of(new LogementDTO(1L, 1L)))
+        .build();
+    proprietaireUpdateDTO = ProprietaireUpdateDTO.builder()
+        .prenom("Karl")
+        .build();
+  }
+
+  @Test
+  public void ProprietaireController_CreerProprietaire_ReturnCreated() throws Exception {
+    given(proprietaireService.creerProprietaire(ArgumentMatchers.any())).willAnswer(
+        invocationOnMock -> proprietaireDTO);
+    when(authService.verifyToken(any(), anyBoolean())).thenReturn(
+        new AbstractMap.SimpleEntry<>(proprietaireDTO.getId(), true)); //skip token verif
+
+    ResultActions response = mockMvc.perform(post("/proprietaires")
+        .header("Authorization",
+            "Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJqb2huZG9lQGdtYWlsLmNvbTs2MjZjMjlhNS0zZGU0LTQ5Y2YtODI4ZS1hNDkxNWQzMzM5N2EiLCJpYXQiOjE3MTQxMTM1NzcsImV4cCI6MTcxNDExNzE3N30.kbU1pVkUHkRXktX44JBFN_xzDv-ZvSmtPjnjORO0vHPiHd3f2MGfDF15VTFO5icIrU_bV9cTqZ70RDlKdlp0-g")
+        .contentType(MediaType.APPLICATION_JSON)
+        .content(objectMapper.writeValueAsString(proprietaireCreationDTO)));
+
+    response.andExpect(MockMvcResultMatchers.status().isCreated())
+        .andExpect(
+            MockMvcResultMatchers.jsonPath("$.nom", CoreMatchers.is(proprietaireDTO.getNom())))
+        .andExpect(MockMvcResultMatchers.jsonPath("$.prenom",
+            CoreMatchers.is(proprietaireDTO.getPrenom())))
+        .andExpect(
+            MockMvcResultMatchers.jsonPath("$.email", CoreMatchers.is(proprietaireDTO.getEmail())))
+        .andExpect(MockMvcResultMatchers.jsonPath("$.numTel",
+            CoreMatchers.is(proprietaireDTO.getNumTel())));
+  }
+
+  @Test
+  public void ProprietaireController_CreerProprietaire_ReturnErrorOwnerInvalidName()
+      throws Exception {
+    proprietaireCreationDTO.setNom("");
+    given(proprietaireService.creerProprietaire(ArgumentMatchers.any())).willAnswer(
+        invocationOnMock -> proprietaireDTO);
+    when(authService.verifyToken(any(), anyBoolean())).thenReturn(
+        new AbstractMap.SimpleEntry<>(proprietaireDTO.getId(), true)); //skip token verif
+
+    ResultActions response = mockMvc.perform(post("/proprietaires")
+        .header("Authorization",
+            "Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJqb2huZG9lQGdtYWlsLmNvbTs2MjZjMjlhNS0zZGU0LTQ5Y2YtODI4ZS1hNDkxNWQzMzM5N2EiLCJpYXQiOjE3MTQxMTM1NzcsImV4cCI6MTcxNDExNzE3N30.kbU1pVkUHkRXktX44JBFN_xzDv-ZvSmtPjnjORO0vHPiHd3f2MGfDF15VTFO5icIrU_bV9cTqZ70RDlKdlp0-g")
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 261).